src/index.jsx

```diff
@@ -1,9 +1,27 @@
-import React from "react";
 import ReactDOM from "react-dom/client";
-import App from "./App.jsx";
+import { createStore } from "redux";
+import { Provider } from "react-redux";
+import reducer from "./store/reducers";
+import TaskBoard from "./components/TaskBoard";
+import NavBar from "./components/NavBar"
 
-ReactDOM.createRoot(document.getElementById("root")).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
+import "./index.scss"
+
+const store = createStore(reducer);
+
+const rootElement = document.getElementById("root");
+const renderApp = () => {
+  const App = (
+    <>
+      <NavBar />
+      <Provider store={store}>
+        <TaskBoard />
+      </Provider>
+    </>
+  );
```

Comment:
Can we move this portion to another part so that it look cleaner? Now its look like messed up.